pilot/pkg/proxy/envoy/v2/ads_test.go

```diff
@@ -23,13 +23,14 @@ import (
 
 	"istio.io/istio/pilot/pkg/model"
 	"istio.io/istio/pilot/pkg/proxy/envoy/v2"
-	"istio.io/istio/tests/util"
+	"istio.io/istio/pkg/test/env"
+	testsUtil "istio.io/istio/tests/util"
```

Comment:
minor nit: naming package `util` is an anti-pattern. It would be better to use something more specific.

Reply:
reverted ... was an artifact of some previous changes.